pkg/updater/updater_test.go

```diff
@@ -4753,10 +4790,11 @@ func TestAlertRow(t *testing.T) {
 				CellIds:  []string{"no", "no again", "very wrong", "yes", "hi", "hello"},
 			},
 			failOpen: 3,
-			expected: alertInfo(3, "no", "very wrong", "no", nil, columns[2], columns[0], columns[3], false),
+			expected: alertInfo(3, "no", "very wrong", "no", nil, columns[2], columns[0], columns[3], false, customColumnHeaders),
```

Comment:
nit: I'd test the custom column headers on 1 or 2 cases, instead of adding it for all cases, just to keep the other test cases a bit clearer.